concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-121	121	the shoulder blade	shoulder blade	172	SHOULDERBLADE	
+Zhang-2019-308-122	122	be single	single	1493	ONE	"It is actually ""one"" in Sino-Tibetan."
+Zhang-2019-308-123	123	six	six	1703	SIX	
+Zhang-2019-308-124	124	the snake brood	snake brood			
+Zhang-2019-308-125	125	the son-in-law	son-in-law	1056	SON-IN-LAW	
+Zhang-2019-308-126	126	be sour	sour	1906	SOUR	
+Zhang-2019-308-127	127	the sovereign, queen	sovereign, queen	1306	QUEEN	
+Zhang-2019-308-128	128	be stiff	stiff			
+Zhang-2019-308-129	129	to strike	strike	353	STRIKE	
+Zhang-2019-308-130	130	the string, cord	string, cord	362	STRING	
+Zhang-2019-308-131	131	be strong, hard	strong, hard	1884	HARD	
+Zhang-2019-308-132	132	to suffer defeat	suffer defeat	1660	LOSE	
+Zhang-2019-308-133	133	the tail	tail	1220	TAIL	
+Zhang-2019-308-134	134	there is no X such that X...	there is no X such that X...			don't know what to do with this one
+Zhang-2019-308-135	135	the thighbone	thighbone	800	THIGH	
+Zhang-2019-308-136	136	be thin	thin	2249	THIN (OF SHAPE OF OBJECT)	
+Zhang-2019-308-137	137	thou	thou	1215	THOU	
+Zhang-2019-308-138	138	three	three	492	THREE	
+Zhang-2019-308-139	139	to beat	to beat	2133	STRIKE OR BEAT	
+Zhang-2019-308-140	140	to bend	to bend	2224	BEND (SOMETHING)	
+Zhang-2019-308-141	141	to break (vi)	to break (vi)	680	BREAK (DESTROY OR GET DESTROYED)	
+Zhang-2019-308-142	142	to break (vt)	to break (vt)	680	BREAK (DESTROY OR GET DESTROYED)	
+Zhang-2019-308-143	143	to carry water on back	to carry water on back			too specific to map
+Zhang-2019-308-144	144	to chase	to chase	1047	PURSUE	
+Zhang-2019-308-145	145	to chew	to chew	321	CHEW	
+Zhang-2019-308-146	146	to confer	to confer			
+Zhang-2019-308-147	147	to curve	to curve	2224	BEND (SOMETHING)	
+Zhang-2019-308-148	148	to defeat	to defeat	782	DEFEAT	
+Zhang-2019-308-149	149	to descend	to descend	2014	GO DOWN (DESCEND)	
+Zhang-2019-308-150	150	to destroy	to destroy	681	DESTROY	
+Zhang-2019-308-151	151	to die	to die	1494	DIE	
+Zhang-2019-308-152	152	to dig out (earth)	to dig out (earth)	1418	DIG	
+Zhang-2019-308-153	153	to drop, drop	to drop, drop	808	DRIP (FALL IN GLOBULES)	
+Zhang-2019-308-154	154	to extinguish	to extinguish	127	EXTINGUISH	
+Zhang-2019-308-155	155	to fall	to fall	1280	FALL	
+Zhang-2019-308-156	156	to fall (from hand)	to fall (from hand)	1280	FALL	
+Zhang-2019-308-157	157	to fry, roast	to fry, roast	2146	ROAST OR FRY	
+Zhang-2019-308-158	158	to give	to give	1447	GIVE	
+Zhang-2019-308-159	159	to hit	to hit	1433	HIT	
+Zhang-2019-308-160	160	to itch	to itch	148	ITCH	
+Zhang-2019-308-161	161	to kill	to kill	1417	KILL	
+Zhang-2019-308-162	162	to lose	to lose	1660	LOSE	
+Zhang-2019-308-163	163	to oppose, disturb	to oppose, disturb	2883	DISTURB	
+Zhang-2019-308-164	164	to peck	to peck	3684	PECK	
+Zhang-2019-308-165	165	to plant	to plant	1486	PLANT (SOMETHING)	
+Zhang-2019-308-166	166	to press	to press	1913	PRESS	
+Zhang-2019-308-167	167	to revive	to revive	1602	WAKE UP	
```

Comment:
WAKE UP seems too specific, judged by the English gloss only. 'revive' refers also to gaining consciousness, I suppose.